emukit/benchmarking/benchmark_plot.py

```diff
@@ -0,0 +1,132 @@
+from itertools import cycle
+from typing import List
+
+import matplotlib.pyplot as plt
```

Comment:
We don't declare an explicit dependency on matplotlib. And neither we should, as emukit can be easily used without plotting. However we probably should create an extra requirement, [similar to GPy](https://github.com/SheffieldML/GPy/blob/devel/setup.py#L167), and maybe call it "benchmarking".

Consequently, I'd suggest we  wrap this import in try/except to handle import error, with similar handling to what we do with model examples at the moment.